app/views/shared/tags/_curated_links_preview.html.erb

```diff
@@ -17,7 +19,13 @@
       <% unless list_item.tagged? %>
         <p class='label label-warning'>Tag was removed</p>
       <% end %>
-      <%= raw list_item.display_title || "<em>Unknown title (#{list_item.base_path})</em>" %>
+      <% if list_item.base_path %>
+        <a href=<%= Plek.new.find('content-tagger') + '/taggings/lookup' + list_item.base_path %>>
+          <%= raw list_item.display_title || "<em>Unknown title (#{list_item.base_path})</em>" %>
```

Comment:
I left `<em>` becuse I've found it there. I can surely replace it with `strong`.